### ISSUE TITLE
Dependent tasks

### ISSUE THREAD

**Issue by farski** [PRX/Porter#22]
Explore the idea of supporting tasks that can work on the result of other tasks, not simply the input file

Eg. Transcode WAV => MP3, then trim the MP3 as one task and encode a lo-fi MP3 as another task from the resultant MP3.

Not sure how much time this saves versus just having all those tasks work directly off the source media in parallel.
